website/content/docs/installation.md

```diff
@@ -39,7 +39,7 @@ Homebrew also automatically installs completions for zsh and Bash.
 Install bloop in other platforms (Windows, Unix, \*bsd) via our python script:
 
 <pre><code class="language-sh">
-$ curl -L https://github.com/scalacenter/bloop/releases/download/<span class="latest-version">1.0.0-M7</span>/install.py | python2
+$ curl -L https://github.com/scalacenter/bloop/releases/download/<span class="latest-version">v1.0.0-M7</span>/install.py | python2
```

Comment:
I just noticed that this won't work because of the JS that fetches the last version from Maven Central. Can you just put the `v` before the `<span>` tag? I think that it should fix it.